debian/control

```diff
@@ -17,10 +17,10 @@ Description: This package contains the Linux integration portion of the Network
 Package: libopx-nas-linux-dev
 Architecture: any
 Depends: ${misc:Depends},libopx-common-dev (>= 1.4.0),libopx-cps-dev (>= 3.6.2),libopx-logging-dev (>= 2.1.0),
-        libopx-nas-common-dev (>= 6.1.0),libopx-base-model-dev (>= 3.148.0),libopx-nas-linux1 (=${binary:Version})
+        libopx-nas-common-dev (>= 6.1.0),libopx-base-model-dev (>= 3.109.0),libopx-nas-linux1 (=${binary:Version})
 Description: This package contains the Linux integration portion of the Network abstractions service.
 
 Package: opx-nas-linux
 Architecture: any
-Depends: ${misc:Depends},opx-cps (>= 3.6.2),python-opx-cps (>= 3.6.2), python-ipaddress (>= 1.0.17-1), python-enum34 (>= 1.1.6-1)
+Depends: ${misc:Depends},opx-cps (>= 3.6.2),python-opx-cps (>= 3.6.2)
```

Comment:
Suggest mentioning these dependency changes in the CHANGELOG and commit message as well.

Reply:
Will revert these changes.